domains/benitaronika.domain.json

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "Project Description",
+    "domain": "is-not-a.dev",
+    "subdomain": "benitaronika",
+
+    "owner": {
+        "repo": "https://github.com/benitaronika/repo",
+        "email": "meghrazicoin@gmail.com"
+    },
+
+    "record": {
+        "A": ["1.1.1.1", "1.0.0.1"],
+        "AAAA": ["::1", "::2"],
+        "CNAME": "benitaronika.com",
+        "MX": ["mx1.benitaronika.com", "mx2.benitaronika.com"],
+        "TXT": ["benitaronika_verification=1234567890"],
+        "CAA": [
+            { "flags": 0, "tag": "issue", "value": "letsencrypt.org" },
+            { "flags": 0, "tag": "issuewild", "value": "sectigo.com" }
+        ],
+        "SRV": [
+            { "priority": 10, "weight": 60, "port": 5060, "target": "sipserver.benitaronika.com" },
+            { "priority": 20, "weight": 10, "port": 5061, "target": "sipbackup.benitaronika.com" }
+        ]
+    },
```

Comment:
Invalid as half of these records are example records. The CNAME cannot be mixed with other records.